docs/release_notes/ibexa_dxp_v4.6.md

```diff
@@ -0,0 +1,466 @@
+
+**Update**: [v4.5.x to v4.6](https://doc.ibexa.co/en/latest/update_and_migration/from_4.5/update_from_4.5/)
+
+## Notable changes
+
+### Ibexa Headless
+
+[[= product_name_content =]] changes name to [[= product_name_headless =]] to emphasize [[= product_name_base =]]'s capacity for headless architecture.
+
+The feature set and capabilities of the product remain the same.
+
+### UX and UI improvements
+
+Several improvements to the Back Office interface enhance the user experience.
+
+#### Customizable dashboard
+
+Users can now customize the dashboard depending on their needs.
+This solution uses an online editor - Dashboard Builder.
+It improves productivity, allows to enhance the default dashboard with additional widgets,
+and helps to make better business decisions based on data.
+
+![Customizable dashboard](img/4.6_customizable_dashboard.png)
+
+For more information, see [Customizable dashboard](https://doc.ibexa.co/projects/userguide/en/master/getting_started/discover_ui/#customizable-dashboard).
+
+#### Page Builder improvements
+
+Page Builder user interface has new functionalities and improvements.
+
+Here are the most important changes:
+
+- new design of Page Builder interface, including block settings window,
+- two main toolboxes: **Elements** and **Structure view**,
+- quick preview of a structure of the page with the possibility of reorganizing the blocks,
+- new visual feedback indicates the correct drop locations,
+- intuitive dragging makes it easier for users to interact with the Page Builder,
+- new actions added in the block settings toolbox,
+- user can now adjust the size of the block settings window,
+- **Undo** and **Redo** buttons.
+
+![Page Builder interface](img/4.6_page_builder_interface.png)
+
+For more information, see [Page Builder interface](https://doc.ibexa.co/projects/userguide/en/master/content_management/create_edit_pages/#page-builder-interface).
+
+#### Editing embedded Content items
+
+User can now edit embedded Content items without leaving current window.
+This function is available in the Rich Text Field when creating Content items, for selected blocks in the Page Builder,
+and while adding or modifying a Content relation.
+
+![Editing embedded Content items](img/4.6_editing_embedded_content_items.png)
+
+For more information, see [Edit embedded Content items](https://doc.ibexa.co/projects/userguide/en/master/content_management/create_edit_content_items/#edit-embedded-content-items).
+
+#### Focus mode
+
+With multiple changes to the Back Office UI intended to expose the most important information and actions, editors can now better focus on their work.
+For more information, see [Focus mode](https://doc.ibexa.co/projects/userguide/en/latest/getting_started/discover_ui/#focus-mode).
+
+![Focus mode](img/4.6_focus_mode.png "Focus mode")
+
+As part of this effort, some other changes were introduced that apply to both regular and Focus mode:
```

Comment:
Since we mention the twig function added for testing remote pim, we shoul mention here the one added to test if focus mode is active or not.